phone/src/i18n.ts

```diff
@@ -3,7 +3,7 @@ import { initReactI18next } from 'react-i18next';
 import en from './locale/en.json';
 import es from './locale/es.json';
 import fr from './locale/fr.json';
-import bh from './locale/bh.json';
+import bh from './locale/bs.json';
```

Comment:
Still incorrect, please change the name of default import and the subsequent reference as well

Reply:
Omg how did I miss it lol

Also needs change in i18n.missingKeys script